### PR TITLE
Adding return code for commands

### PR DESCRIPTION
**Pull request by scolandrea**
Hey, I figured out the commands are now returning an exit code.

### CODE REVIEW

**Comment by DavidLambauer** [comment]
Hi! Thank you for the PR. We'll look into this shortly!

File: src/Console/Command/ListMetricsCommand.php [hunk 1]
```diff
@@ -72,5 +72,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $table->render();
+
+        return Command::SUCCESS;
```
Comment (VladyslavSikailo) [inline src/Console/Command/ListMetricsCommand.php:76]: Could you please use `\Magento\Framework\Console\Cli::RETURN_SUCCESS` here?

File: src/Console/Command/CollectMetricsCommand.php [hunk 2]
```diff
@@ -60,5 +60,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         } else {
             $this->aggregateMetricsCron->execute();
         }
+
+        return Command::SUCCESS;
```
Comment (VladyslavSikailo) [inline src/Console/Command/CollectMetricsCommand.php:64]: Could you please use `\Magento\Framework\Console\Cli::RETURN_SUCCESS` here?

**Review by VladyslavSikailo — CHANGES_REQUESTED** [review]
Hi @scolandrea

Thank you for the improvements to the module!

There is a minor change request - please take a look. And also, may I ask you to the `int` as a return type for `execute` methods?

We appreciate your interest!
Thanks

File: src/Console/Command/ListMetricsCommand.php (hunk 1 above)
Comment (scolandrea) [reply src/Console/Command/ListMetricsCommand.php]: Done!

File: src/Console/Command/CollectMetricsCommand.php (hunk 2 above)
Comment (scolandrea) [reply src/Console/Command/CollectMetricsCommand.php]: Done!

**Pull request merged**